src/components/App.js

```diff
@@ -1,5 +1,15 @@
 import React, { useState, useEffect } from 'react';
 
+import {
+  BrowserRouter as Router,
+  Route,
+  Switch
+} from 'react-router-dom';
+
+/* import { Navbar } from './Navbar'; */
+
+import { Navbar, Product, Allproducts } from './';
+
 import {
   getSomething
```

Comment:
Remove dead code. Make sure that you are using descriptive names for functions and whatnot.